models/perceptron.py

```diff
@@ -6,19 +6,16 @@
 import theano.tensor as T
 
 from smartpy import Model
-from utils import load_dict_from_json_file, save_dict_to_json_file
-from utils import WeightsInitializer
-
-
-dtype = theano.config.floatX
+from utils import load_dict_from_json_file, save_dict_to_json_file, WeightsInitializer
```

Comment:
If we intend to pickle the models, that'd be unnecessary as we won't use 
json for long.

Adam Salvail
Doctorant en informatique
Université de Sherbrooke
819 821-8000 (62038)

On 21/07/15 04:29 PM, Marc-Alexandre Côté wrote:
>
> In models/perceptron.py 
> <https://github.com/SMART-Lab/models/pull/6#discussion_r35150322>:
>
> > @@ -6,19 +6,16 @@
> >  import theano.tensor as T
> >
> >  from smartpy import Model
> > -from utils import load_dict_from_json_file, save_dict_to_json_file
> > -from utils import WeightsInitializer
> > -
> > -
> > -dtype = theano.config.floatX
> > +from utils import load_dict_from_json_file, save_dict_to_json_file, WeightsInitializer
>
> Should we load |load_dict_from_json_file| and |save_dict_to_json_file| 
> from smartpy's utils?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/SMART-Lab/models/pull/6/files#r35150322>.Web Bug 
> from 
> https://github.com/notifications/beacon/AEJelD3d9BpMOhcDJz3PwkN36-pMEzW4ks5ofqNGgaJpZM4FdAYQ.gif
>